src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -153,14 +143,14 @@ export class ServerConnectionIndicator extends LitElement {
       // based on when the animation loop started
       const elapsedAnimationMS = Date.now() - this.animationStartMS;
 
-      const remainingAnimationMS =
-        ANIMATION_DURATION_MS -
-        (elapsedAnimationMS %
-          // while the animation is reversed, the animation delay
-          // is included in the total play time
-          (this.animationState === ServerConnectionState.DISCONNECTING
-            ? ANIMATION_DURATION_MS + ANIMATION_DELAY_MS
-            : ANIMATION_DURATION_MS));
+      // while the animation is reversed, the animation delay
+      // is included in the total play time
```

Comment:
Proper punctuation and capitalization: https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar

```suggestion
      // While the animation is reversed, the animation delay
      // is included in the total play time.
```